pkg/api/controllers/apikey/apikey.go

```diff
@@ -17,20 +21,30 @@ import (
 //	@Summary		List API keys
 //	@Description	List API keys
 //	@Produce		json
-//	@Success		200	{array}	ApiKey
+//	@Success		200	{array}	ApiKeyViewDTO
 //	@Router			/apikey [get]
 //
 //	@id				ListClientApiKeys
 func ListClientApiKeys(ctx *gin.Context) {
 	server := server.GetInstance(nil)
 
+	currentApiKeyName, err := server.ApiKeyService.GetApiKeyName(ctx.Request.Context(), util.ExtractToken(ctx))
+	if err != nil {
+		ctx.AbortWithError(http.StatusNotFound, fmt.Errorf("failed to get current api key name: %w", err))
```

Comment:
Remove the abort here. We don't want to abort if the current api key can not be found. That's not very important. If, for some reason, the key can not be found, then none of the returned api keys will be the Current which is fine